### PR TITLE
Add a new description for OTS

### PR DESCRIPTION
**Pull request by Ivoz**
Addresses #94 

I've written up a new description. I thought others might wish to comment on whether they like it rather than pushing it straight in.

Any comments on the language? Cheers.

### CODE REVIEW

File: _includes/_head.html [hunk 1]
```diff
@@ -4,36 +4,36 @@
...
+  volunteer-driven organisation providing programming and tech-literacy 
+  workshops, learning materials and gatherings for our surrounding 
+  communities. We organise meetups the world over designed to allow 
+  anyone to take control of the technology they use, in a friendly and 
+  safe environment. Everyone, no matter their experience or background 
+  is invited to come and participate and learn with us. Complete beginners 
+  and experienced professionals alike are welcome to join and contribute!">
 
-  <meta name="description" content="OpenTechSchool is a community initiative to
-  provide programming and other tech-related workshops to women and their
```
Comment (lehmannro) [inline _includes/_head.html:15]: I really liked that first sentence as it is very catchy.

File: _includes/_head.html [hunk 2]
```diff
@@ -4,36 +4,36 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 
   <title>OpenTechSchool – {{ page.title }}</title>
   <link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' 
-  rel='stylesheet' type='text/css' />
+  rel='stylesheet' type='text/css'>
+  <meta name="description" content="OpenTechSchool is an open, 
+  volunteer-driven organisation providing programming and tech-literacy 
+  workshops, learning materials and gatherings for our surrounding 
+  communities. We organise meetups the world over designed to allow
```
Comment (lehmannro) [inline _includes/_head.html:16]: Too many successive verbs.

File: _includes/_head.html [hunk 3]
```diff
@@ -4,36 +4,36 @@
...
 
   <title>OpenTechSchool – {{ page.title }}</title>
   <link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' 
-  rel='stylesheet' type='text/css' />
+  rel='stylesheet' type='text/css'>
+  <meta name="description" content="OpenTechSchool is an open, 
+  volunteer-driven organisation providing programming and tech-literacy 
+  workshops, learning materials and gatherings for our surrounding 
+  communities. We organise meetups the world over designed to allow 
+  anyone to take control of the technology they use, in a friendly and
```
Comment (lehmannro) [inline _includes/_head.html:17]: This sounds like the friendly and safe environment is an afterthought and taking control over something is our first goal.  I think a safe and friendly environment has always been our firstmost priority.

**Comment by Ivoz** [comment]
Ben has suggested maybe this could come up on the global meet, which ain't a bad idea

**Comment by lehmannro** [comment]
Stefan noticed deviating from the about section in the `<meta>` tag is maybe not too useful.  At the Global Review Meeting we proposed working from the current blurb, and just changing minor bits which have changed.  Beatrice has, in the meantime, changed the Facebook description to be gender-inclusive.

**Pull request closed without merging**